src/Components/Slider/Slide.js

```diff
@@ -1,54 +1,60 @@
-import PropTypes from 'prop-types';
+import React from 'react'
 
-import { SlideItem, InfoContainer, Header, Description } from './styled';
+import PropTypes from 'prop-types'
 
-const Slide = ({ width, ...props }) => {
+import { SlideItem, InfoContainer, Header, Description } from './styled'
 
-    if(props.data){
+const Slide = ({ id, width, ...props }) => {
+    if (props.data) {
         const {
             title,
-            description:[
-                {text}
+            // Not sure if in the next two cases it's ok to disable the eslint rule
+            // eslint-disable-next-line react/prop-types
```

Comment:
This is eslint telling you to declare the prop that you are extracting from data. Otherwise your component won't know which type of data to expect for this prop.